CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # HEAD
 
+# [v2.3.1](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/releases/tag/v2.3.1)
+
+Image updates:
+- Fix an issue in block devices cleanup by Kubernetes Job
+  ([#60](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/pull/60))
+
+Deployment updates:
+- Support pod security policy
```

Comment:
Do any of these deployment updates depend on newer than 1.12?

Reply:
no